### ISSUE TITLE
onlyRunRules ignored on master

### ISSUE THREAD

**Issue by andrewgaul** [lewisd32/lint-maven-plugin#23]
Release 0.0.8 honors these but master ignores them.

**Comment by twohey** [comment]
It appears that 0.0.9 has this bug, which makes it challenging to use for my needs

**Comment by lewisd32** [comment]
I've been having some trouble getting this to build on my development computer at home, so unfortunately I'm not sure when I'll be able to look at this. PR's welcome though.

**Comment by lkwg82** [comment]
can you post your configuration plz

**Comment by lewisd32** [comment]
This affects release 0.0.9.

**Comment by twohey** [comment]
Cutting down my whole project is a bit much, but I can show my plugin configuration:

```xml
      <plugin>
        <groupId>com.lewisd</groupId>
        <artifactId>lint-maven-plugin</artifactId>
        <version>0.0.9</version>
        <configuration>
          <failOnViolation>true</failOnViolation>
          <onlyRunRules>
            <rule>ExecutionId</rule>
            <rule>DotVersionProperty</rule>
            <rule>GAVOrder</rule>
            <rule>RedundantDepVersion</rule>
            <rule>RedundantPluginVersion</rule>
            <rule>ProfileOnlyAddModules</rule>
            <rule>DuplicateDep</rule>
            <rule>OSSInceptionYearRule</rule>
          </onlyRunRules>
        </configuration>
        <executions>
          <execution>
            <id>lint-maven</id>
            <phase>validate</phase>
            <goals>
              <goal>check</goal>
            </goals>
          </execution>
        </executions>
      </plugin>
```

**Comment by lkwg82** [comment]
I swa the same and will take a look into that.

**Comment by lkwg82** [comment]
This fix needs a release, would you do that?!.

**Comment by lewisd32** [comment]
Oh, yes, I'll try to do that shortly.

**Comment by lewisd32** [comment]
I've released 0.0.10.  It should filter into Maven Central in the next while.

**Comment by lewisd32** [comment]
@twohey Did this resolve the issue you were experiencing?
